inbm/cloudadapter-agent/cloudadapter/utilities.py

```diff
@@ -8,11 +8,14 @@
 import logging
 import os
 
-from .constants import SLEEP_DELAY, UCC_FILE, UCC_ENABLED_FLAG
+from .constants import SLEEP_DELAY, UCC_FILE, UCC_ENABLED_FLAG, UCC_CLIENT_ID
 from threading import Thread, Event
 from typing import Any, Callable
 
 logger = logging.getLogger(__name__)
+
+MAX_CLIENT_ID_LENGTH = 12
```

Comment:
Please increase the max length size to atleast 36.
Typical UUID length being 36 chars.

Do send a note to UCC team on this limitation and see if they are OK with the 36 char limit.

Reply:
@pillaiabhishek This PR isn't done yet and open for review.  But I thought they were going to use the Mac address which would be 12 characters as they described.